Formula/vcluster.rb

```diff
@@ -0,0 +1,35 @@
+class Vcluster < Formula
+  desc "Creates fully functional virtual k8s cluster inside host k8s cluster's namespace"
+  homepage "https://www.vcluster.com"
+  url "https://github.com/loft-sh/vcluster.git",
+    tag:      "v0.7.1",
+    revision: "dc0ff6f96e9c96fe2caa77e79c2dffc921b4fd49"
```

Comment:
```suggestion
  url "https://github.com/loft-sh/vcluster.git",
      tag:      "v0.7.1",
      revision: "dc0ff6f96e9c96fe2caa77e79c2dffc921b4fd49"
```
Align the arguments to `url`, please.

Reply:
Some of the formules has similar pattern so I thought this is the preferred  way. 
https://github.com/Homebrew/homebrew-core/blob/master/Formula/skaffold.rb